### PR TITLE
Create scenarios Post and get for Entities

### PR DESCRIPTION
**Pull request by jpgonzales27**

### CODE REVIEW

File: src/main/java/api/ApiManager.java [hunk 1]
```diff
@@ -12,6 +15,7 @@
     private static int status;
 
     public static ApiResponse execute(ApiRequest apiRequest) {
+        RestAssured.filters(new RequestLoggingFilter(), new ResponseLoggingFilter());
```
Comment (GustavoZHA) [inline src/main/java/api/ApiManager.java]: Delete this line is not necessary

File: src/test/java/AccountTest.java [hunk 2]
```diff
@@ -33,19 +33,19 @@ public void get_all_account_successful_200() {
...
 
     @Test(groups = {"deleteAccount"})
     public void createProjectWithStatusCode200() throws JsonProcessingException{
-        Account accountTemp = new Account();
-        accountTemp.setName("Account41");
+        Account accountTemp = new Account("Account41");
         ApiRequest apiRequest = baseRequest().method(ApiMethod.POST).endpoint(ParameterEndPoints.ACCOUNT)
                 .body(new ObjectMapper().writeValueAsString(accountTemp)).build();
         ApiResponse response = ApiManager.execute(apiRequest);
+        response.getResponse().then().log().all();
```
Comment (GustavoZHA) [inline src/test/java/AccountTest.java]: Is not necessary to show the log yet

File: src/test/java/BaseTestAccount.java [hunk 3]
```diff
@@ -33,15 +33,14 @@
     public IBuilderApiRequest baseRequest() {
         ApiResponse response = ApiManager.executeToken();
         return new ApiRequestBuilder()
-                .baseUri(ParameterEndPoints.URL_BASEO)
+                .baseUri(ParameterEndPoints.URL_BASE)
                 .headers("Authorization", "Bearer " + response.getBody(Token.class).getAccess_token());
     }
```
Comment (GustavoZHA) [inline src/test/java/BaseTestAccount.java:39]: Delete bank line

Comment (jpgonzales27) [reply src/test/java/BaseTestAccount.java:39]: Ok gus i ready solved

**Pull request merged**